src/main/resources/l10n/JabRef_de.properties

```diff
@@ -2251,9 +2251,9 @@ There_was_one_file_that_could_not_be_imported.=Eine_Datei_konnte_nicht_importier
 There_were_%0_files_which_could_not_be_imported.=%0_Dateien_konnten_nicht_importiert_werden.
 
 Migration_help_information=Hilfe_zur_Migration
-Entered_library_has_obsolete_structure_and_is_no_longer_supported.=Die_gewählte_Datenbank_nutzt_eine_veraltete,_nicht_mehr_unterstützte_Struktur.
```

Comment:
This and the two others are shared database ones. See https://github.com/JabRef/jabref/pull/2748.